source/LogbookPanel.cpp

```diff
@@ -31,6 +34,8 @@ PARTICULAR PURPOSE.  See the GNU General Public License for more details.
 
 #include <algorithm>
 #include <set>
+#include <utility>
+#include <iostream>
```

Comment:
I think you forgot to remove this, as well as `#include "Files.h"`

Also you already included utility in the header, no need to include it twice; I suppose its needed for the std::pair

Reply:
Yeah, those are from some debugging.

Reply:
Yeah, those are from some debugging.